dist/qs.js

```diff
@@ -324,9 +324,12 @@ var arrayPrefixGenerators = {
     indices: function indices(prefix, key) {
         return prefix + '[' + key + ']';
     },
+    pipe: 'pipe',
```

Comment:
please revert all changes to this file; it's only updated at release time.

Reply:
> If we're going to add these options to stringify, we'd need them in `parse` as well, so things can round trip.

@ljharb No problem, I'll make that change. Thank you again for taking the time to review this PR!

Reply:
Will do 👍